mpu6050/python/mpu6050Python/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN apt-get -q -y update && \
     python3-venv \
 # DO NOT REMOVE THIS LABEL: this is used for VS Code automation
     # __torizon_packages_prod_start__
+	python3-smbus2:arm64 \
```

Comment:
This is added automatically by the extension, so it can be removed of the commit here